lang/syn/src/codegen/accounts/mod.rs

```diff
@@ -30,6 +33,8 @@ pub fn generate(accs: &AccountsStruct) -> proc_macro2::TokenStream {
 
         #__client_accounts_mod
         #__cpi_client_accounts_mod
+
+        #bumps_struct
```

Comment:
Nit: it would be better if we could move this line above the private client code(`#__client_accounts_mod`).